Makefile

```diff
@@ -33,7 +33,16 @@ test:
 
 .PHONY: func-test
 func-test:
-	CGO_ENABLED=1 go test -coverpkg ./pkg/... -coverprofile coverage.out -timeout 180m ./tests/...
+	# we run the tests in our tests directory and for every package listed additionally
+	# some of the newer tests (starting with the refactored CloudDNS tests) can run as
+	# either unit or integration tests, using mocks of the API for unit tests. They need
+	# the --integration-tests flag to run as integration tests.
```

Comment:
flag looks different below

Reply:
I wanted to point out that go does not differentiate between `--integration-tests` and `-integration-tests`, but the flag is not actually plural - good catch x)